_i18n/ar/resources/user-guides/make-payment.md

```diff
@@ -0,0 +1,80 @@
+{% include disclaimer.html translated="no" translationOutdated="no" %}
+
+## How to make a payment
+
+### What is important to know about Monero payments?
+
+- **Irreversible**: Monero transactions are irreversible once they are confirmed (included in a @block) by a miner. If you've sent a transaction and it was confirmed by a miner, the only way to get your money back is to ask the receiver to send it back to you.
+
+- **Private**: people watching the @blockchain will not know important details of your transaction, like sender address (your address), amount and recipient address. The recipient of your payment will know all details of the transaction, except the sender address (your wallet address) and transaction description.
+
+- **Transaction fee**: the sender pays a transaction fee to incentivize miners to include the transaction in a block, unlike credit cards, where the merchant pays the transaction fee.
+
+### How to make a payment using Monero GUI
+
+1) Go to `Send` page. In order to send a transaction, your node and wallet must be synchronized and the currently selected account must have an available (unlocked) balance.
+
+<img src="/img/resources/user-guides/en/make-payment/send_page.png" style="width: 600px;"/>
+
+2) On `Address` field, enter the recipient address. If it's a Monero address, it should start with `4...` or `8...`.
+
+Alternatively you can import an address from a QR code by clicking on `Grab QR code from screen` and `Import from address book` buttons.
+
+You can also paste in this field a Monero payment request with an amount request, which starts with `monero:...`.
+
+This field also supports OpenAlias addresses, which are human readable addresses (for instance, `donate.monero.cc`). If you enter an OpenAlias address, a `Resolve` button will be displayed. Click on it to retrieve its Monero address.
```

Comment:
@openalias, we have a Moneropedia entry for it.

Reply:
We should use donate.getmonero.org to match the contributing page, here and below.